packages/devtools_app/lib/src/screens/logging/_log_details.dart

```diff
@@ -45,12 +43,12 @@ class _LogDetailsState extends State<LogDetails>
 
   Future<void> _computeLogDetails() async {
     if (widget.log?.needsComputing ?? false) {
-      await widget.log.compute();
+      await widget.log!.compute();
       setState(() {});
     }
   }
 
-  bool showSimple(LogData log) => log != null && !log.needsComputing;
+  bool showSimple(LogData log) => !log.needsComputing;
```

Comment:
looks like this method can be deleted. It is not used anywhere

Reply:
done